### ISSUE TITLE
use Steps with tilte is a Link, when Link clicked, error occurs

### ISSUE THREAD

**Issue by ChinaXing** [ant-design/ant-design#5501]
#### Environment(required)

- antd version: 2.8.1
- OS and its version: macOS Sierra 10.12.3
- Browser and its version: chrom canary , 59.0.3049.0

#### What did you do? Please provide steps to re-produce your problem.

Use Steps , and it's title is ReactNode : <Link to='...' /> imports from react-router,

when I click the tilte, it will produce the following errors in console:

```
Uncaught Error: findDOMNode was called on an unmounted component.
    at invariant (invariant.js:44)
    at Object.findDOMNode (findDOMNode.js:53)
    at Steps._this.calcStepOffsetWidth (Steps.js:48)
    at invokeFunc (index.js:160)
    at trailingEdge (index.js:207)
    at timerExpired (index.js:195)
```

#### What do you expected?

there is not error should produce

#### What happen?

```
Uncaught Error: findDOMNode was called on an unmounted component.
    at invariant (invariant.js:44)
    at Object.findDOMNode (findDOMNode.js:53)
    at Steps._this.calcStepOffsetWidth (Steps.js:48)
    at invokeFunc (index.js:160)
    at trailingEdge (index.js:207)
    at timerExpired (index.js:195)
```

**Comment by afc163** [comment]
Could you provide a re-producible demo to help us, by forking and editing this codepen: http://codepen.io/benjycui/pen/KgPZrE?editors=001

(A good and minimum codpen will save a lot of time 😃)

**Comment by afc163** [comment]
Fixed in rc-steps@2.4.3, retry reinstall node_modules.
